sources/install/package_ad.sh

```diff
@@ -97,7 +98,7 @@ function install_crackmapexec() {
     git -C /opt/tools/ clone --depth 1 https://github.com/Porchetta-Industries/CrackMapExec
     pipx install /opt/tools/CrackMapExec/
     mkdir -p ~/.cme
-    [ -f ~/.cme/cme.conf ] && mv ~/.cme/cme.conf ~/.cme/cme.conf.bak
+    [[ -f "~/.cme/cme.conf" ]] && mv ~/.cme/cme.conf ~/.cme/cme.conf.bak
```

Comment:
Tilde doesn't expand in quotes. Quotes need to be removed.

Reply:
Done